src/Header/MainManager.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef Group4_Header_MainManager_H_
+#define Group4_Header_MainManager_H_
+
+#include <netinet/in.h>
+
+#include <vector>
+#include <unordered_map>
+
+#include "HttpListener.h"
+#include "RequestHandler.h"
+
+
+
+//will initialize http listener, do multiprocess when listener callback
+class MainManager
+{
+public:
+    //httpListener callback, do fork and new requestHandler to call api, child process add to parents vector for managing.
+    void NewRequest(const string &requestStr);
```

Comment:
Please use underscore for naming: `request_str`
https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
Done